#Will contain a speed and angle for each wheel of the car(meaning eight values total)
#Sent from "turn_radius"
#Worked by: Lutu

#probably a better way to do this with like arrays of 0-3 or something but whatever, this will work for now

float64 angleFrontLeft
float64 angleFrontRight
float64 angleRearLeft
float64 angleRearRight

float64 speedFrontLeft
float64 speedFrontRight
float64 speedRearLeft
float64 speedRearRight
